package.json

```diff
@@ -46,7 +45,7 @@
     "ember-elsewhere": "0.3.0",
     "ember-engines": "~0.3.4",
     "ember-export-application-global": "^1.0.5",
-    "ember-frost-core": "0.29.0",
+    "ember-frost-core": "^1.0.0",
     "ember-hook": "^1.3.1",
     "ember-load-initializers": "^0.5.1",
     "ember-one-way-controls": "0.9.2",
```

Comment:
We also need to remove `perfect-scrollbar` from `bower.json`

Note: same comment also valid for `notifier`